projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -39,9 +39,7 @@ PACKER_OVA_VAR_FILES+=$(PACKER_OVA_CONF_FILE)
 
 PACKER_RAW_VAR_FILES?=
 PACKER_CLOUDSTACK_VAR_FILES?=
-
-PACKER_NUTANIX_CONF_FILES=$(MAKE_ROOT)/packer/nutanix/nutanix.json
-PACKER_NUTANIX_VAR_FILES+=$(PACKER_NUTANIX_CONF_FILES)
+PACKER_NUTANIX_VAR_FILES?=
```

Comment:
we made this change after bumping to the latest image-builder release tag because it started validated the nutanix settings. I was curious about why this started and if this change would cause issues.  How are you avoiding the validation error now without this change?

Reply:
We're providing nutanix settings to AWS image-builder with --nutanix-config option. nutanix-config is json file with the same params like in nutanix.json

For example:
`
$ cat ./nutanix-connection-rhel-9.json
{
  "nutanix_cluster_name": "e2e***",
  "source_image_name": "rhel-9.2-x86_64-kvm.qcow2",
  "image_url": "http://data.***/rhel-9.2-x86_64-kvm.qcow2",
  "image_name": "eksa-rhel-9-kube-1-27",
  "nutanix_subnet_name": "vlan***",
  "nutanix_endpoint": "prism-ncn-ci.***",
  "nutanix_insecure": "false",
  "nutanix_port": "9440",
  "nutanix_username": "***",
  "nutanix_password": "***",
  "rhel_username": "***",
  "rhel_password": "****"
}
`

JFYI: I've tested it with the following command:
`
CODEBUILD_CI=true CODEBUILD_SRC_DIR=/***/eks-anywhere-build-tooling REPO_NO_CLONE=true ./image-builder build --os redhat --os-version 9 --hypervisor nutanix --release-channel 1-27 --nutanix-config ./nutanix-connection-rhel-9.json -v 20
`

Reply:
yeah thats kind of what  I figured. Its possible its just a presubmit issue. let me enable the tests and lets see what it does.